src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/CommunityApiClient.kt

```diff
@@ -69,6 +71,13 @@ class CommunityApiClient(@Qualifier("communityWebClientAppScope") private val we
       .bodyToMono(Requirements::class.java)
       .block()?.requirements ?: listOf()
   }
+
+  fun updateTier(tierDto: TierDto, crn: String): ResponseEntity<Void>? {
+    return webClient
+      .post()
+      .uri("offenders/crn/$crn/tier/${tierDto.protectLevel}${tierDto.changeLevel.ordinal}")
+      .retrieve().toBodilessEntity().block()
```

Comment:
This endpoint does return the offender but I suppose we aren't interested in that